### PR TITLE
:arrow_up: update xunit dependency

### PR DESCRIPTION
**Pull request by sebastianMindee**
## Types of changes

- [x] Docs change / refactoring / dependency upgrade
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires a change to the official Guide documentation.

### CODE REVIEW

File: .github/workflows/integration-tests.yml [hunk 1]
```diff
@@ -14,7 +14,7 @@ env:
 jobs:
   test-nix:
     name: Integration Test on *NIX
-    timeout-minutes: 30
+    timeout-minutes: 10
```
Comment (ianardee) [inline .github/workflows/integration-tests.yml:17]: no reason to change this

File: .github/workflows/integration-tests.yml [hunk 2]
```diff
@@ -54,7 +54,7 @@ jobs:
   test-windows:
     name: Integration Test on Windows
     needs: test-nix
-    timeout-minutes: 30
+    timeout-minutes: 10
```
Comment (ianardee) [inline .github/workflows/integration-tests.yml:57]: no reason to change this

**Review by ianardee — APPROVED** [review]

File: .github/workflows/integration-tests.yml (hunk 1 above)
Comment (sebastianMindee) [reply .github/workflows/integration-tests.yml:17]: Yeah that one is true

File: .github/workflows/integration-tests.yml (hunk 2 above)
Comment (sebastianMindee) [reply .github/workflows/integration-tests.yml:57]: I'm getting a bit tired of having to wait 30 minutes to see if the 472 run crashes. Have you got any better ideas?

**Review by ianardee — APPROVED** [review]

File: .github/workflows/integration-tests.yml (hunk 2 above)
Comment (ianardee) [reply .github/workflows/integration-tests.yml:57]: it usually crashes before the 30 min is up for me :shrug: 
anyway approved

**Pull request merged**